src/server.js

```diff
@@ -119,8 +129,10 @@ initDb(config.database).then(db => {
   })
 
   server.plugins.openapi.setHost(server.info.host + ':' + server.info.port)
-  server.log(['info'], `Server running on ${server.info.host}:${server.info.port}`)
+  server.log(['info'], `Server running`)
```

Comment:
please revert this line to the pervious one

Reply:
Reverted.